importer_utils.py

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/python
+# -*- coding: utf-8  -*-
+import datetime
+import json
+import re
+import pywikibot
+import os
+from wikidataStuff.WikidataStuff import WikidataStuff as wds
+
+site_cache = {}
+
+DATA_DIRECTORY = "data"
+
+
+def load_json(filename):
+    try:
+        with open(filename) as f:
+            try:
+                return json.load(f)
+            except ValueError:
+                print("Failed to decode file {}.".format(filename))
+    except OSError:
+        print("File {} does not exist.".format(filename))
+
+
+def json_to_file(filename, json_content):
+    with open(filename, 'w') as f:
+        json.dump(json_content, f, sort_keys=True,
+                  indent=4,
+                  ensure_ascii=False,
+                  default=datetime_convert)
+
+
+def append_line_to_file(text, filename):
+    with open(filename, 'a') as f:
+        f.write(text + "\n")
+
+
+def get_current_timestamp():
+    return datetime.datetime.now().strftime('%Y-%m-%d_%H:%M:%S')
+
+
+def string_is_q_item(text):
+    """Check if a string looks like a WD ID."""
+    pattern = re.compile("^Q[0-9]+$", re.I)
+    try:
+        m = pattern.match(text)
+    except TypeError:
+        return False
+    if m:
+        return True
+    else:
+        return False
+
+
+def datetime_convert(dt_object):
+    if isinstance(dt_object, datetime.datetime):
+        return dt_object.__str__()
+
+
+def wd_template(template_type, value):
+    return "{{" + template_type + "|" + value + "}}"
+
+
+def dict_to_iso_date(date_dict):
+    """
+    Convert pywikiboty-style date dictionary
+    to ISO string ("2002-10-23").
+
+    @param date_dict: dictionary like
+    {"year" : 2002, "month" : 10, "day" : 23}
+    """
+    iso_date = ""
+    if "year" in date_dict:
+        iso_date += str(date_dict["year"])
+    if "month" in date_dict:
+        iso_date += "-" + str(date_dict["month"])
+    if "day" in date_dict:
+        iso_date += "-" + str(date_dict["day"])
+    return iso_date
+
+
+def create_site_instance(language, family):
+    """Create an instance of a Wiki site (convenience function)."""
+    site_key = (language, family)
+    site = site_cache.get(site_key)
+    if not site:
+        site = pywikibot.Site(language, family)
+        site_cache[site_key] = site
+    return site
+
+
+def is_vowel(char):
+    vowels = "auoiyéeöåäáæø"
+    if char.lower() in vowels:
+        return True
+    else:
+        return False
+
+
+def get_last_char(text):
+    """Return the last character of string."""
+    return text[-1]
+
+
+def last_char_is_vowel(text):
+    """Check if last char of string is vowel."""
+    return is_vowel(get_last_char(text))
+
+
+def date_to_dict(datestring, dateformat):
```

Comment:
Added a docstring.

I thought about doing it, but it means you have to initiate Wikidatastuff and it feels heavy. It's also preferable to know what's going on in here, since it's a simple thing.